Makefile

```diff
@@ -55,18 +65,21 @@ endif
 	  sed -e s:@localstatedir@:$(localstatedir): > scripts/liquidsoap.logrotate
 
 .PHONY: doc-install api-doc-install
-doc-install:
-	$(MAKE) -C doc doc-install
+doc-install: doc
+	if [ "$(OS_TYPE)" != "Win32" ]; then \
+	  $(MAKE) -C doc $@; \
+	fi
 api-doc-install:
-	$(V)echo Installing developer documentation...
+	$(V)echo "Installing developer documentation..."
 	$(V)$(INSTALL) -d $(datadir)/doc/$(DISTDIR)/api
 	$(V)for doc in $(wildcard autodoc/liquidsoap/*.html autodoc/liquidsoap/*.css) ;\
 	do $(INSTALL_DATA) $$doc $(datadir)/doc/$(DISTDIR)/api ; \
 	done
 
 # user and group are defined in Makefile.defs, written by configure.
-
-install-local: doc-install
+.PHONY: install dist print-tarball-filename tarball
+install: doc-install
```

Comment:
I still think that doc install should be optional. Otherwise, everyone compiling liquidsoap will need to have `pandoc` installed to be able to install the build.